classes/Rest/Controllers/WarehouseControllerProvider.php

```diff
@@ -1761,17 +1768,26 @@ private function processHistoryRequest(Application $app, XDUser $user, $realm, $
      */
     private function processHistoryDefaultRealmRequest(Application $app, $action)
     {
+        $config = \Xdmod\Config::factory();
+
+        $rawstats = $config['rawstatistics'];
+
+        $results = array();
+
+        if (isset($rawstats['realms'])) {
+            foreach($rawstats['realms'] as $realm => $realmconfig) {
+                $results[] = array('dtype' => 'realm',
```

Comment:
This should be on the next line
```suggestion
                $results[] = array(
                    'dtype' => 'realm',
```